frontend/src/lib/i18n/utils/assertTranslationKey.ts

```diff
@@ -0,0 +1,9 @@
+import type {TranslationKey} from '$types';
+
+/**
+ * Asserts blindly that key is `TranslationKey` for use with translation keys that are formed dynamically.
+ * TODO: In the future, check that the key is a valid translation key.
+ */
+export function assertTranslationKey(key: string): TranslationKey {
```

Comment:
Used to tell the linter that dynamically keys are valid in Svelte html code where we cannot use the `as` keyword.